src/modules/commander/Commander.cpp

```diff
@@ -359,6 +360,25 @@ int Commander::custom_command(int argc, char *argv[])
 		return (ret ? 0 : 1);
 	}
 
+	if (!strcmp(argv[0], "origin")) {
```

Comment:
Maybe "set origin" or "set global origin"?

Reply:
I settled on `set_ekf_origin`, let me know if you'd prefer it to be either of your suggestions, or feel free to just change it to anything you'd like!

Reply:
I think that works and might help reduce confusion with the home position.